dungeon-crawler/android_mobile/src/main/java/com/novoda/dungeoncrawler/MainActivity.java

```diff
@@ -40,10 +50,23 @@ protected void onResume() {
         Log.d("TUT", "Game starting");
     }
 
+    @Override
+    public void onPauseGame() {
+        Toast.makeText(this, "Game Paused", Toast.LENGTH_SHORT).show();
+        game.onPauseGame();
```

Comment:
I think this should be `game.pause()` you want to tell it to pause, not inform it that the game has been paused? (which makees less sense right?)

Reply:
Done in aba87dcbcf9576d27a95a4eef5d2011f5525ee53